doc/source/quick_start.md

```diff
@@ -46,6 +56,11 @@ fn_glacier_outlines = xdem.examples.get_path("longyearbyen_glacier_outlines")
 print(f"DEM 1: {fn_dem_ref}, \nDEM 2: {fn_dem_tba}, \nOutlines: {fn_glacier_outlines}")
 ```
 
+```{tip}
+:class: margin
+Set up your {ref}`verbosity` to manage outputs to the console (or a file) during execution!
+```
+
```

Comment:
Just a question on line 78 below, which was not edited. 
I assume this is just for the example, as both DEMs are originally on the same vertical datum? It must impact coregistration a bit... I think it's great to showcase this functionality, but it woul dbe good to have a sample data set with actual vertical datum difference.

Reply:
+ L 119: "An basic" -> "a basic"

Reply:
Good catch. And opened https://github.com/GlacioHack/xdem/issues/635 for the vertical datum of the DEM.